ui/imports/shared/popups/AddEditSavedAddressPopup.qml

```diff
@@ -100,6 +100,7 @@ StatusDialog {
             }
             readOnly: root.edit || root.addAddress
             wrongInputValidationError: qsTr("Please enter a valid ENS name OR Ethereum Address")
+            ownAddressError: qsTr("Can't add yourself as a saved address")
```

Comment:
The component actually affecting this bug is the one under `ui/app/AppLayouts/Wallet/popups/`. I thought I'd replicate the fix here as well for consistency, but I can revert this if desired.
BTW, do we want to merge these two into a single shared resource that fits all uses? They seem to be almost identical.

Reply:
Definitely go for one (shared) component; duplication is bad :)